src/main/scala/microsites/Exceptions.scala

```diff
@@ -0,0 +1,23 @@
+package microsites
```

Comment:
:art: apache header

Reply:
They are going to be generated in the next PR, automatically by `ci-docs` :)